StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Controllers/PayWithLinkViewController.swift

```diff
@@ -103,7 +103,11 @@ final class PayWithLinkViewController: UINavigationController {
     private var accountContext: LinkAccountContext = .shared
 
     private var linkAccount: PaymentSheetLinkAccount? {
-        get { accountContext.account }
+        get {
+            let account = accountContext.account
+            account?.paymentSheetLinkAccountDelegate = self
+            return account
```

Comment:
Sorry, i'm not following why this is needed.

Reply:
The `PaymentSheetLinkAccount` needs to be able to communicate with the `PayWithLinkViewController` to inform it of updates. So when a `linkAccount` is set, the PayWithLinkViewController needs to set itself as the PaymentSheetLinkAccount's delegate. But maybe I can make this cleaner.

Reply:
Maybe this is clearer? https://github.com/stripe/stripe-ios/pull/4432/commits/b4e379716d7b10204086f8fdff2c1c3572b89e3c#diff-7d5339851903df687dc841f3deb388ba566a2f0b319c57c38671cb529b80e7a4R185

Reply:
If we add onAccountChange Observer, then seems like we can revert the this change back to:
```
get { accountContext.account }
```
?

Reply:
Ah yes, I did but accidentally stashed. Pushing that!